dynamics-nav/Statements-and-Expressions.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Statements and Expressions"
+description: Describes Statements and Expressions, examples and typical expressions.
```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.